third_party/move/move-vm/types/src/values/values_impl.rs

```diff
@@ -404,6 +411,27 @@ impl Container {
         }
     }
 
+    fn get_enum_tag(&self) -> PartialVMResult<u16> {
+        match self {
+            Self::Struct(vals) => {
+                let vals = vals.borrow();
+                vals.first()
+                    .and_then(|v| match v {
+                        Value::U16(x) => Some(*x),
+                        _ => None,
+                    })
+                    .ok_or_else(|| {
+                        PartialVMError::new(StatusCode::UNKNOWN_INVARIANT_VIOLATION_ERROR)
+                            .with_sub_status(EINDEXED_REF_TAG_MISMATCH)
+                    })
+            },
+            _ => Err(
```

Comment:
Is there a reason we changed this from a fully enumerated list (which is typically preferred) to an "_" catch all?